external/buildscripts/build.pl

```diff
@@ -772,19 +769,11 @@
 
 		my $androidNdkRoot = $ENV{ANDROID_NDK_ROOT};
 		my $androidPlatformRoot = "$androidNdkRoot/platforms/$ENV{ANDROID_PLATFORM}/arch-$platformRootPostfix";
-		my $androidToolchain = "$androidNdkRoot/toolchains/$toolchainName/prebuilt/$ENV{HOST_ENV}";
 
-		if (!(-d "$androidToolchain"))
-		{
-			if (-d "$androidToolchain-x86")
-			{
-				$androidToolchain = "$androidToolchain-x86";
-			}
-			else
-			{
-				$androidToolchain = "$androidToolchain-x86_64";
-			}
-		}
+		my $androidToolchain = "$androidNdkRoot/toolchains/$hostTriple-clang";
```

Comment:
When looking at NDK r16b directory structure I can't see this path. Shouldn't it be "$androidNdkRoot/toolchains/llvm/prebuilt/$ENV{HOST_ENV}"?

Reply:
This tripped me up for a while. I think that you need to generate this directory using `build/tools/make_standalone_toolchain.py` (see the code just below this). I discovered this on this documentation page: https://developer.android.com/ndk/guides/standalone_toolchain

Once I did this, things started to work. Maybe I'm incorrect though?

Reply:
Other Unity projects don't call this script and are configured to consume compiler/headers/whatever directly without copying them to "install" directory. This works too though.